src/pages/Home/AlkemioActivitySection.tsx

```diff
@@ -54,7 +54,7 @@ const AlkemioActivitySection: FC<{
   return (
     <DashboardActivitySection
       headerText={t('pages.activity.title', { blockName: 'Platform' })}
-      bodyText={t('pages.activity.summary', { blockName: 'Alkemio Hub' })}
```

Comment:
remove this label from the translation file if not used elsewhere.

Reply:
good spot, removed